app/src/action_handler/actions/proposal/submit.rs

```diff
@@ -78,7 +78,8 @@ impl ActionHandler for ProposalSubmit {
                 use penumbra_transaction::plan::ActionPlan::*;
                 for action in &transaction_plan.actions {
                     match action {
-                        Spend(_) | Output(_) | Swap(_) | SwapClaim(_) | DelegatorVote(_) => {
+                        Spend(_) | Output(_) | Swap(_) | SwapClaim(_) | DelegatorVote(_)
+                        | WithdrawalPlan(_) => {
```

Comment:
wait, why do we want to ban the DAO from making withdrawal plans? it seems potentially useful to be able to send funds directly from a proposal to, e.g., a multisig contract on another ibc chain.

i don't think the withdrawal plan requires any proving, so it seems like it shouldn't be a dos risk